pyPhin.py

```diff
@@ -109,47 +118,80 @@ def verify(self, contact, verifyUrl, deviceUUID, verificationCode):
 		'''
 		req = requests.get(
 			self.baseUrl+locationUrl,
-			headers=self.createHeader(authToken, deviceUUID, "2.0.1")
+			headers=self.createHeader(deviceUUID, authToken, "2.0.1")
```

Comment:
what is this version used for? Maybe put in a constant?

Reply:
Each page has a different version associated with it.